npm/Dockerfile

```diff
@@ -1,6 +1,24 @@
-# Use a minimal image as a parent image
+# Build npm
+FROM golang:1.16-alpine AS builder
+# Build args
+ARG VERSION
+ARG NPM_AI_PATH
+ARG NPM_AI_ID
+
+WORKDIR /usr/local/src/npm
+
+# Copy the source
+COPY . .
+
+# Build npm
+RUN CGO_ENABLED=0 go build -v -o /usr/local/bin/azure-npm -ldflags "-X main.version="$VERSION" -X "$NPM_AI_PATH"="$NPM_AI_ID"" -gcflags="-dwarflocationlists=true" npm/cmd/*.go
+
+# Use a minimal image as a final image base
 FROM ubuntu:focal
```

Comment:
probably should be @vakalapa:
a whole ubuntu image seems a little heavy to just get net-utils, any reason this couldn't be alpine or smaller?

Reply:
I didn't try it, just copied what was there before, I can try alpine

Reply:
Alpine failed the cyclonus tests

Reply:
We can look at this in another PR but for this scope I'm going to leave it with `ubuntu:focal`
alpine results:
![image](https://user-images.githubusercontent.com/35265851/134745012-100d0e2e-93b9-498c-974f-3d9712b4c2fb.png)